controllers/openshift_controller.go

```diff
@@ -218,6 +239,37 @@ func (r *KataConfigOpenShiftReconciler) processDaemonsetForMonitor() *appsv1.Dae
 	}
 }
 
+func (r *KataConfigOpenShiftReconciler) processDashboardConfigMap() *corev1.ConfigMap {
+
+	r.Log.Info("Creating sandboxed containers dashboard in the OpenShift console")
+	cmName := "grafana-dashboard-sandboxed-containers"
+	cmLabels := map[string]string{
+		"console.openshift.io/dashboard": "true",
+	}
+	cmNamespace := "openshift-config-managed"
+
+	// retrieve content of the dashboard from our own namespace
+	foundCm := &corev1.ConfigMap{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: "openshift-sandboxed-containers-operator"}, foundCm)
+	if err != nil {
+		r.Log.Error(err, "could not get dashboard data")
+		//return nil
+	}
+
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "apps/v1",
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cmName,
+			Namespace: cmNamespace,
```

Comment:
So no additional RBAC permission necessary to be able to do this after all?  There is much to learn... ;-)

Reply:
I was surprised too. I launched it for testing, thinking "ok, I'll have an error", and it just worked :-)
Now I didn't check what permissions we would need, compared to what we already have. Maybe we actually had it already for another reason?
For instance, we have get/update permissions on the "namespaces" resources... Would that cover it?

Reply:
ok, hold on : I realize that in order to install the daemonset, I'm still using the workaround to give "privileged" scc to the operator... when we actually use the required (limited) privileges there, maybe this will just stop working?

Reply:
I think you could do this because of the privileged SCC. The latest code does have the RBAC rules for namespace updates. But I'm not sure if there is a specific RBAC rule required for configMap creation in different namespace.
Can you try with the latest operator code which sets the SCC ? You don't need to manually add the SCC to the service account.

Reply:
IIUC the syntax of ClusterRole `rules:` stanza doesn't allow you to specify a particular namespace for which a permission is valid (although it does have syntax for particular `resourceNames`).  Looking at the current contents of `config/rbac/role.yaml` though, it seems that the permission to create, delete, get, list, patch, update and watch configmaps (among other resource types) is already granted which might explain why you didn't have to add it (should have verified this earlier, sorry!).

Or it's the SCC as Pradipta says. :-)

Reply:
Ok, so I can confirm it works WITHOUT manually adding the scc.
I used to do that out of habit during my test process, but since I rebased from master, I don't need it anymore - the daemonset and configmap are installed properly, without touching anything...
... it just works ! :-)